src/Api/ResourceApiController.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+namespace Saritasa\LaravelControllers\Api;
+
+use Dingo\Api\Http\Request;
+use Dingo\Api\Http\Response;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Validation\ValidationException;
+use Saritasa\DingoApi\Traits\PaginatedOutput;
+use Saritasa\Enums\PagingType;
+use Saritasa\Exceptions\InvalidEnumValueException;
+use Saritasa\LaravelEntityServices\Contracts\IEntityService;
+use Saritasa\LaravelEntityServices\Contracts\IEntityServiceFactory;
+use Saritasa\LaravelEntityServices\Exceptions\EntityServiceException;
+use Saritasa\LaravelEntityServices\Exceptions\EntityServiceOperationException;
+use Saritasa\LaravelRepositories\DTO\SortOptions;
+use Saritasa\Transformers\IDataTransformer;
+
+/**
+ * Default controller to handle CRUD operation with models.
+ */
+class ResourceApiController extends BaseApiController
+{
+    use PaginatedOutput;
+
+    /**
+     * Entities services factory.
```

Comment:
I can get this description by splitting variable name by words. As a result this description has no value. Can't we describe why this factory need?